tests/001_tests.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from django.db.models.query import QuerySet
+from users.models import Profile
+
+
+@pytest.mark.django_db(transaction=True)
+class TestProfile:
+    def test_profie_created(cls):
+        out = Profile.objects.all()
+        assert isinstance(out, QuerySet)
+        assert all(isinstance(p, Profile) for p in out)
+        profiles_names = []
+        for p in out:
+            profiles_names.append(p.user.username)
+        assert profiles_names == [
+            ('Test-user-comments1'),
+            ('Test-user-comments2'),
+            ('Test-user-profile3'),
+            ('Test-user-profile4')
```

Comment:
in the generate function you only generate test-user-comments-1/2, so where are these created?

Reply:
+1 this test should test commenting capability

Reply:
Test-user-profile3, Test-user-profile4 are created in the users migrations test data
Test-user-comments1,  Test-user-comments2 are created in the commenting_system migrations test data

Reply:
Correct, thanks